### PR TITLE
To datepicker

### PR DESCRIPTION
**Pull request by ckjones818**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -64,5 +64,8 @@ group :production do
   gem 'pg'
 end
 
-# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
+# Windows does not include zoneinfo files, so bundle the tzinfo-data gem\
+gem 'bootstrap'
```
Comment (lilyosah) [inline Gemfile:68]: Do we need bootstrap? @ckjones818

Comment (ckjones818) [reply Gemfile:68]: What's in this PR is all I did -- I didn't add anything else or manually install any bootstrap. The plugin seems to use some bootstrap functionality but installation was just a pretty standard add to gemfile/bundle install/add require statements process so I assume it's all internal

Comment (ckjones818) [reply Gemfile:68]: @lilyosah

File: spec/features/calendar_feature_spec.rb [hunk 2]
```diff
@@ -48,7 +48,6 @@
     # visit root_path
     wait_for_ajax
     expect(page.find('#calendar')).to have_content("Task2")
-    expect(Task.find_by(title:'Task2').start.to_date.to_formatted_s(:db)).to eq(DateTime.now.to_date.to_formatted_s(:db))
```
Comment (lilyosah) [inline spec/features/calendar_feature_spec.rb:51]: Why was this removed? @ckjones818
